en, zh: add release notes for v1.2.1 (#1359)
pingcap/docs-tidb-operator: en/releases/release-1.2.1.md

---
title: TiDB Operator 1.2.1 Release Notes
---

# TiDB Operator 1.2.1 Release Notes

Release date: August 18, 2021

TiDB Operator version: 1.2.1

## Rolling update changes

> - If you have enabled `hostNetwork` for TiCDC, after upgrading the TiDB Operator, the TiCDC Pod will roll update due to [#4141](https://github.com/pingcap/tidb-operator/pull/4141)

Author: recreation 感觉不太能体现 roll upgrade

> - If [`hostNetwork`](../configure-a-tidb-cluster.md#hostnetwork) is enabled for TiCDC, upgrading TiDB Operator will cause the recreation of the TiCDC Pod due to [#4141](https://github.com/pingcap/tidb-operator/pull/4141)

Author: ```suggestion
- If [`hostNetwork`](../configure-a-tidb-cluster.md#hostnetwork) is enabled for TiCDC, upgrading TiDB Operator will cause rolling upgrade of the TiCDC Pod due to [#4141](https://github.com/pingcap/tidb-operator/pull/4141)
```

Author: @qiancai  PTAL

Reviewer: 中文里是删除重建，删除重建  = rolling upgrade?

Author: emm，是删除重建，过程是滚动的

Author: 看了下以前的，感觉不用改了 💯

## Improvements

> - Support `hostNetwork` for all components ([#4141](https://github.com/pingcap/tidb-operator/pull/4141), [@DanielZhangQD](https://github.com/DanielZhangQD)

Reviewer 2: ```suggestion
- Support `hostNetwork` for all components in TidbCluster ([#4141](https://github.com/pingcap/tidb-operator/pull/4141), [@DanielZhangQD](https://github.com/DanielZhangQD)
```

> - Support configuring [`hostNetwork`](../configure-a-tidb-cluster.md#hostnetwork) for all components in TidbCluster so that all components can use host network ([#4141](https://github.com/pingcap/tidb-operator/pull/4141), [@DanielZhangQD](https://github.com/DanielZhangQD)

Reviewer: ```suggestion
- Support configuring [`hostNetwork`](../configure-a-tidb-cluster.md#hostnetwork) for all components in TidbCluster so that all components can use host network ([#4141](https://github.com/pingcap/tidb-operator/pull/4141), [@DanielZhangQD](https://github.com/DanielZhangQD))
```
